data/src/main/java/com/android/developers/androidify/data/GeminiNanoGenerationDataSource.kt

```diff
@@ -60,6 +61,28 @@ internal class GeminiNanoGenerationDataSourceImpl @Inject constructor(
         return response.candidates[0].text
     }
 
+    override suspend fun validatePromptHasEnoughInformation(inputPrompt: String): ValidatedDescription? {
+        if (!downloader.isModelDownloaded()) return null
+
+        val response = downloader.generativeModel.generateContent(
+            generateContentRequest(
+                TextPart(remoteConfigDataSource.promptTextVerifyNano() + inputPrompt),
+            ) {
+                temperature = 0.0f
+                maxOutputTokens = 256
```

Comment:
should this value be something smaller like 2 since the output is either 'invalid' or 'valid'?